src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,8 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
+        return userService.findByEmail(email) != null
+                && userService.findByEmail(email).getPassword().equals(password);
```

Comment:
save `userService.findByEmail(email)` to the variable and work with it, to avoid duplicate method calling